2019-10-21T12:25:43.5297003+01:00;VSTestExecutionThread;Info;SpecFlow+Runner execution started
2019-10-21T12:25:43.5297003+01:00;VSTestExecutionThread;Info;SpecFlow+Runner 3.0.391 in Framework clr40 in x86 mode execution started
2019-10-21T12:25:43.5297003+01:00;VSTestExecutionThread;Info;TestAdapter Location: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\packages\SpecRun.Runner.3.0.391\tools\net45\TechTalk.SpecRun.VisualStudio.TestAdapter.dll
2019-10-21T12:25:43.5996991+01:00;VSTestExecutionThread;Info;SpecRun: running tests in C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T12:25:43.6007038+01:00;VSTestExecutionThread;Info;Output folder configured to "C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults" (default)
2019-10-21T12:25:43.6007038+01:00;VSTestExecutionThread;Info;Profile: 
2019-10-21T12:25:43.6357008+01:00;VSTestExecutionThread;Info;Log file path: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults\Unnamed project__2019-10-21T122543.log
2019-10-21T12:25:43.6527024+01:00;VSTestExecutionThread;Info;SpecFlow+Runner Evaluation Mode: Please purchase at http://www.specflow.org/plus to remove test execution delay.
2019-10-21T12:25:43.6607002+01:00;Thread#14;Info;Checking license
2019-10-21T12:25:43.6857015+01:00;Thread#14;Info;Evaluation mode: test execution starts in 5 seconds... (this delay is removed in the registered version)
2019-10-21T12:25:48.6878575+01:00;Thread#14;Info;Evaluation delay finished.
2019-10-21T12:25:48.6938576+01:00;Thread#14;Info;License check completed
2019-10-21T12:25:48.6938576+01:00;Thread#14;Info;starting test run
2019-10-21T12:25:48.7068615+01:00;Thread#14;Info;Discovering target: Default
2019-10-21T12:25:48.7078604+01:00;Thread#14;Info;Discovering tests from assembly: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T12:25:49.2138743+01:00:Appdomain Disposed
2019-10-21T12:25:49.2138743+01:00;Thread#14;Info;Discovered tests from assembly: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T12:25:49.2158698+01:00:Test Suite Dump
2019-10-21T12:25:49.2188689+01:00:#0: Unnamed project/ - 
2019-10-21T12:25:49.2188689+01:00:#1: Default - 
2019-10-21T12:25:49.2188689+01:00:#2: CSharpSpecFlowSelenium - 
2019-10-21T12:25:49.2188689+01:00:#3: BBCLogin1 - 
2019-10-21T12:25:49.2628660+01:00:#4: Invalid password - TestAssembly:CSharpSpecFlowSelenium/Feature:BBCLogin1/Scenario:Invalid+password
2019-10-21T12:25:49.2628660+01:00:#5: Valid password - TestAssembly:CSharpSpecFlowSelenium/Feature:BBCLogin1/Scenario:Valid+password
2019-10-21T12:25:49.2628660+01:00:#6: Valid Signout - TestAssembly:CSharpSpecFlowSelenium/Feature:BBCLogin1/Scenario:Valid+Signout
2019-10-21T12:25:49.2628660+01:00:End: Test Suite Dump
2019-10-21T12:25:49.2868713+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T12:25:49.2938685+01:00:Thread#:Scheduler: testitem #4 from normal test queue 
2019-10-21T12:25:49.2938685+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T12:25:49.2988693+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T12:25:49.2988693+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T12:25:49.2988693+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T12:25:49.2988693+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T12:25:49.3008689+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T12:25:49.3028662+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-10-21T12:25:49.3068722+01:00:Thread #0: creating new test thread executor for CSharpSpecFlowSelenium, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null (target: Default)
2019-10-21T12:25:49.3088685+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-10-21T12:25:49.3098669+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-10-21T12:25:49.3109122+01:00:Thread #0: using TechTalk.SpecRun.Framework.Execution.InProcess.InProcAppDomainTestAssemblyExecutorHost to execute tests
2019-10-21T12:25:49.4048673+01:00:Thread#0: Test #4/0 is started on thread #0
2019-10-21T12:25:49.5658801+01:00:Thread#0:T:-> Loading plugin C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\SpecRun.Runtime.SpecFlowPlugin.dll
2019-10-21T12:25:49.5739107+01:00:Thread#0:T:-> Using default config
2019-10-21T12:25:52.9379017+01:00:Thread#0:B:Given I am on the login page
2019-10-21T12:25:53.0439033+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T12:25:53.0449032+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T12:25:53.0459024+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:53.0459024+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T12:25:53.0459024+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:53.0469016+01:00:Thread#0:B:When I press login
2019-10-21T12:25:53.0479041+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:53.0509050+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T12:25:53.0519026+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:53.1789045+01:00:Thread#0:TestItem#4:Scheduler: reschedule tests for testitem #4
2019-10-21T12:25:53.1849077+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T12:25:53.1849077+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-10-21T12:25:53.1849077+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T12:25:53.1849077+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T12:25:53.1849077+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T12:25:53.1849077+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T12:25:53.1849077+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T12:25:53.1849077+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T12:25:53.1849077+01:00:Thread#0:Scheduler: with testItemExecutionIndex 1
2019-10-21T12:25:53.6979130+01:00:Thread#0: Test #4/0 is finished on thread #0 as Failed within 4.2930457 seconds
2019-10-21T12:25:53.7009105+01:00:Thread#0: Test #4/1 is started on thread #0
2019-10-21T12:25:53.7019087+01:00:Thread#0:T:warning: The previous ScenarioContext was not disposed.
2019-10-21T12:25:56.8967667+01:00:Thread#0:B:Given I am on the login page
2019-10-21T12:25:56.8987677+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T12:25:56.8997669+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T12:25:56.8997669+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:56.8997669+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T12:25:56.8997669+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:56.9007662+01:00:Thread#0:B:When I press login
2019-10-21T12:25:56.9007662+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:56.9007662+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T12:25:56.9007662+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:25:56.9017837+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T12:25:56.9017837+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-10-21T12:25:56.9017837+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T12:25:56.9017837+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T12:25:56.9017837+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T12:25:56.9017837+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T12:25:56.9017837+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T12:25:56.9017837+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T12:25:56.9017837+01:00:Thread#0:Scheduler: with testItemExecutionIndex 2
2019-10-21T12:25:57.3227693+01:00:Thread#0: Test #4/1 is finished on thread #0 as Failed within 3.6218588 seconds
2019-10-21T12:25:57.3227693+01:00:Thread#0: Test #4/2 is started on thread #0
2019-10-21T12:25:57.3227693+01:00:Thread#0:T:warning: The previous ScenarioContext was not disposed.
2019-10-21T12:26:00.3233428+01:00:Thread#0:B:Given I am on the login page
2019-10-21T12:26:00.3253379+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T12:26:00.3253379+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T12:26:00.3253379+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:26:00.3253379+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T12:26:00.3263379+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:26:00.3263379+01:00:Thread#0:B:When I press login
2019-10-21T12:26:00.3263379+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:26:00.3263379+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T12:26:00.3263379+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:26:00.3273376+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T12:26:00.3273376+01:00:Thread#:Scheduler: testitem #5 from normal test queue 
2019-10-21T12:26:00.3273376+01:00:Thread#0:Scheduler: considering testitem #5 for thread 0
2019-10-21T12:26:00.3273376+01:00:Thread#0:TestItem#5:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T12:26:00.3273376+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadSpecific:False
2019-10-21T12:26:00.3273376+01:00:Thread#0:TestItem#5:runsOnOneOtherThreadUnspecific:False
2019-10-21T12:26:00.3273376+01:00:Thread#0:Scheduler: using testitem #5 for thread 0
2019-10-21T12:26:00.3273376+01:00:Thread#0:Scheduler: Test #5 is scheduled for thread
2019-10-21T12:26:00.3273376+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-10-21T12:26:00.7513422+01:00:Thread#0: Test #4/2 is finished on thread #0 as Failed within 3.4285729 seconds
2019-10-21T12:26:00.7513422+01:00:Thread#0: Test #5/0 is started on thread #0
2019-10-21T12:26:00.7513422+01:00:Thread#0:T:warning: The previous FeatureContext was not disposed.
2019-10-21T12:26:00.7523455+01:00:Thread#0:T:warning: The previous ScenarioContext was not disposed.
2019-10-21T12:26:03.9067432+01:00:Thread#0:B:Given I am on the bbc login page
2019-10-21T12:26:05.6087576+01:00:Thread#0:T:done: BBCLogin1Steps.GivenIAmOnTheBbcLoginPage() (1.7s)
2019-10-21T12:26:05.6087576+01:00:Thread#0:B:And I have entered a valid username
2019-10-21T12:26:06.2577666+01:00:Thread#0:T:done: BBCLogin1Steps.GivenIHaveEnteredAValidUsername() (0.6s)
2019-10-21T12:26:06.2577666+01:00:Thread#0:B:And I have entered a valid password
2019-10-21T12:26:06.6387701+01:00:Thread#0:T:done: BBCLogin1Steps.GivenIHaveEnteredAValidPassword() (0.4s)
2019-10-21T12:26:06.6387701+01:00:Thread#0:B:When I press login btn
2019-10-21T12:26:09.9707993+01:00:Thread#0:T:done: BBCLogin1Steps.WhenIPressLoginBtn() (3.3s)
2019-10-21T12:26:09.9707993+01:00:Thread#0:B:Then I should see the user account homepage
2019-10-21T12:26:10.5558060+01:00:Thread#0:T:done: BBCLogin1Steps.ThenIShouldSeeTheUserAccountHomepage() (0.6s)
2019-10-21T12:26:10.5728039+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T12:26:10.5728039+01:00:Thread#:Scheduler: testitem #6 from normal test queue 
2019-10-21T12:26:10.5728039+01:00:Thread#0:Scheduler: considering testitem #6 for thread 0
2019-10-21T12:26:10.5728039+01:00:Thread#0:TestItem#6:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T12:26:10.5728039+01:00:Thread#0:TestItem#6:runsOnOneOtherThreadSpecific:False
2019-10-21T12:26:10.5728039+01:00:Thread#0:TestItem#6:runsOnOneOtherThreadUnspecific:False
2019-10-21T12:26:10.5728039+01:00:Thread#0:Scheduler: using testitem #6 for thread 0
2019-10-21T12:26:10.5728039+01:00:Thread#0:Scheduler: Test #6 is scheduled for thread
2019-10-21T12:26:10.5738053+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-10-21T12:26:10.9758217+01:00:Thread#0: Test #5/0 is finished on thread #0 as CleanupFailed within 10.2244795 seconds
2019-10-21T12:26:10.9838115+01:00:Thread#0: Test #6/0 is started on thread #0
2019-10-21T12:26:10.9958130+01:00:Thread#0:T:warning: The previous FeatureContext was not disposed.
2019-10-21T12:26:11.0028108+01:00:Thread#0:T:warning: The previous ScenarioContext was not disposed.
2019-10-21T12:26:13.6028372+01:00:Thread#0:B:Given I am on the bbc account page
2019-10-21T12:26:13.6128392+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T12:26:13.6198366+01:00:Thread#0:B:When I press the signout btn
2019-10-21T12:26:13.6278361+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:26:13.6348351+01:00:Thread#0:B:Then I should see the signout page displayed
2019-10-21T12:26:13.6408379+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T12:26:13.6618374+01:00:Cancelling because configured number of failures reached
2019-10-21T12:26:14.1958438+01:00:Thread#0: Test #6/0 is finished on thread #0 as Failed within 3.2120323 seconds
2019-10-21T12:26:14.2498409+01:00:Appdomain Disposed
2019-10-21T12:26:14.2528429+01:00:Trying to cancel thread
2019-10-21T12:26:14.2528429+01:00:Thread cancelled
2019-10-21T12:26:14.2648433+01:00:Thread#-1:Scheduler: get next test for thread -1
2019-10-21T12:26:14.2658411+01:00:Thread#:Scheduler: at the end of normal test queue
2019-10-21T12:26:14.2658411+01:00:Thread#-1:Scheduler: enumerator for thread is at end
2019-10-21T12:26:14.2748441+01:00;Thread#14;Info;test run finished
2019-10-21T12:26:14.2758420+01:00;Thread#14;Info;publishing test results
2019-10-21T12:26:14.2988435+01:00;Thread#14;Info;test results published
2019-10-21T12:26:14.2998430+01:00;Thread#14;Info;generating reports
2019-10-21T12:26:14.5878475+01:00;Thread#14;Info;Starting external report generation process
2019-10-21T12:26:17.5768708+01:00;Thread#14;Info;Successfully generated reports.
2019-10-21T12:26:17.5818723+01:00;Thread#14;Info;Successfully generated reports
2019-10-21T12:26:17.5938752+01:00:Total: 3 (test executions: 5)
2019-10-21T12:26:17.5958711+01:00:Succeeded: 0
2019-10-21T12:26:17.5968720+01:00:Ignored: 0
2019-10-21T12:26:17.5968720+01:00:Pending: 0
2019-10-21T12:26:17.5968720+01:00:Skipped: 0
2019-10-21T12:26:17.5978701+01:00:Failed: 3
2019-10-21T12:26:17.6148764+01:00;VSTestExecutionThread;Info;Adding attachments to VSTest
2019-10-21T12:26:17.6148764+01:00;VSTestExecutionThread;Info;adding Test Execution Report 'C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults\Unnamed project__2019-10-21T122543.html'
========== End of log file ==========
